templates/themes/horizon/invenio_app_rdm/site_footer.html

```diff
@@ -0,0 +1,45 @@
+{#
+    -*- coding: utf-8 -*-
+    Copyright (C) 2023-2024 CERN.
+
+    Invenio App RDM is free software; you can redistribute it and/or modify it
+    under the terms of the MIT License; see LICENSE file for more details.
+#}
+
+<div class="footer-bottom sub-footer theme-default rel-p-1">
+  <div class="ui container">
```

Comment:
How is it different from this [file](https://github.com/inveniosoftware/invenio-app-rdm/blob/e9f55834d9869756b325f871562cb384a739faba/invenio_app_rdm/theme/templates/themes/default/invenio_app_rdm/site_footer.html)?

Reply:
https://github.com/zenodo/zenodo-rdm/blob/f764f3b704669b2d4b3766a0275923025102ad34/templates/themes/horizon/invenio_app_rdm/site_footer.html#L16